From 4fed51358288e646b82f4fd598435f125f985177 Mon Sep 17 00:00:00 2001
From: ta264 <ta264@users.noreply.github.com>
Date: Fri, 26 Feb 2021 08:42:34 +0000
Subject: [PATCH] FreeBSD support

---
 Directory.Build.props  | 1 +
 eng/Dependencies.props | 1 +
 2 files changed, 2 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index f2f54bab5d..cd41aba973 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -163,6 +163,7 @@
       win-arm64;
       osx-x64;
       osx-arm64;
+      freebsd-x64;
       linux-musl-x64;
       linux-musl-arm;
       linux-musl-arm64;
diff --git a/eng/Dependencies.props b/eng/Dependencies.props
index 4858ba89c5..853b455267 100644
--- a/eng/Dependencies.props
+++ b/eng/Dependencies.props
@@ -91,6 +91,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.win-arm64" />
     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.osx-x64" />
     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.osx-arm64" />
+    <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.freebsd-x64" />
     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.linux-x64" />
     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.linux-arm" />
     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.linux-arm64" />
-- 
2.30.0

